src/lib/radio/radio.ts

```diff
@@ -160,6 +177,21 @@ export class MdRadioGroup extends _MdRadioGroupMixinBase
     this._checkSelectedRadioButton();
   }
 
+  /** Whether the radio group is diabled */
+  @Input()
+  get disabled() { return this._disabled; }
+  set disabled(value) {
+    this._disabled = value;
+    if (this._radios) {
+      // Update radios disabled state
+      this._radios.forEach((r) => r._markForCheck());
+    }
+  }
+
+  constructor(private _change: ChangeDetectorRef) {
```

Comment:
I would call this `_changeDetector`, since `_change` could be confused with the change event